src/js/components/Button/StyledButton.js

```diff
@@ -34,9 +41,24 @@ const primaryStyle = props => css`
     props.theme,
     props.theme.button.color,
   )}
-  border-radius: ${props.theme.button.border.radius};
+  border-radius: ${radiusStyle(props)};
 `;
 
+const sizeStyle = props => {
+  const size = props.size || 'medium';
+  const data = props.theme.button.size[size];
```

Comment:
I'm wondering if we should connect this more directly to Text sizing in the theme, rather than allowing Button to have different text sizing? We should discuss with the designers.

Reply:
I think you are right.

Reply:
Done.